<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGeneralControl</class>
 <widget class="QMainWindow" name="FormGeneralControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>440</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Main State Machine</string>
   </property>
   <widget class="QSimpleShape" name="ledON">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>24</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledOpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledOFF">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>24</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledStandby">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>24</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenloop">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Loop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOFF">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStandby">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Standby</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnON">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledDC">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>64</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDC">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>DC</string>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledWave">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>64</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWaveform">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Waveform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInterlock">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Force Interlock</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="titleGroupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>440</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTuning">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuning</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWaveforms">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Waveform Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFirmware">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameters</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>440</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Waveform State Machine</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>54</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Trigger</string>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledSemiAuto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>24</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledAuto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>54</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QSimpleShape" name="ledManual">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>84</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="ledType">
     <enum>QSimpleShape::Status</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkip">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Waveform Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Waveform Cycle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSemiauto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Semiauto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Auto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Manual">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTrigger">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Trigger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResume">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QENumericEdit" name="txtStart">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="leadingZeros">
     <number>0</number>
    </property>
    <property name="precision">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>4.000000000000000</double>
    </property>
   </widget>
   <widget class="QENumericEdit" name="txtCycle">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="leadingZeros">
     <number>0</number>
    </property>
    <property name="precision">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>4.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reference Value</string>
   </property>
  </widget>
  <widget class="QENumericEdit" name="txtReference">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="variable" stdset="0">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="precision">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="addUnits">
    <bool>false</bool>
   </property>
   <property name="writeOnChange">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSet">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
